python/services/github_service.py

```diff
@@ -376,6 +396,35 @@ def get_paginated_list_of_team_repositories(self, team_name: str, after_cursor:
             "after_cursor": after_cursor
         })
 
+    def fetch_all_repositories_in_org(self) -> list[dict[str, Any]]:
+        """A wrapper function to run a GraphQL query to get the list of repo names in the organisation
+
+        Returns:
+            list: A list of the organisation repos names
+        """
+        has_next_page = True
+        after_cursor = None
+        repos = []
+
+        while has_next_page:
+            data = self.get_paginated_list_of_repositories(after_cursor)
+
+            # Retrieve the name of the repos
```

Comment:
Think it'd be cleaner to wrap the code in some named functions over the comments? 

It'd also allow us to reuse this functionality if there's more use cases for it